VS 21 February/VS 21 February/Program.cs

```diff
@@ -1,102 +1,227 @@
 ﻿using System;
-
+using Microsoft.VisualBasic;
+using System.ComponentModel;
+using VS_21_February.Task7.autopark;
 class Hello
 {
     static void Main(string[] args)
     {
-        //First Home Task
-        VS_21_February.Phone iphone_13 = new VS_21_February.Phone();
+        //HomeTask 1 lesson 7
+        //Public autopark
+        Bus bus_1 = new Bus("Minsk - Gomel", "2290-NB-3", "14:30", 30);
+        Trolleybus trolleybus_1 = new Trolleybus("DS Kyntsevshina - DS Malinovka", "9917-VS-5", "12:25", 83);
+        Tramcar tramcar_1 = new Tramcar("DS Zeleny Lyg - Myasnikova", "7760-AZ-6", "08:17", 100);
+        Bus bus_2 = new Bus("Minsk - Berlin", "7777-NN-7", "06:20", 28);
+        Trolleybus trolleybus_2 = new Trolleybus("DS Kyntsevshina - DS Malinovka", "7717-VS-6", "14:15", 70);
+
+        //Interface IGetInfo
+        bus_1.Main();
+        trolleybus_1.Main();
+        tramcar_1.Main();
```

Comment:
Methods should be a verb
example - GetInfo();

Reply:
Got it - fixed! Thank you!